src/pages/nft/NftInformation.tsx

```diff
@@ -0,0 +1,253 @@
+        <>
+          <img className="token-logo" src={Neo2} alt="token-logo" />
+          <span>NEO LEGACY</span>
+        </>
+      )
+    } else {
+      return (
+        <>
+          <span>NEO LEGACY</span>
+        </>
+      )
+    }
+  }
+
+  return (
+    <div id="Nft" className="page-container">
+      {/*
+          BackButton won't work for now
+      */}
+      <BackButton url="" text="back to address information" />
+      <div className="inner-page-container">
+        <div className="title-container">
+          {ROUTES.NFT.renderIcon()}
+          <h1>NFT Information</h1>
+        </div>
+
+        <div className="nft-title">
+          <span>{nftState.value?.name}</span>
+          <span className="id">
+            ID:
+            <span className="number">
+              {truncateText(isMobileOrTablet, nftState.value?.id)}
+            </span>
+          </span>
+        </div>
+
+        {!nftState.isLoading && (
+          <div className="nft-information-container">
+            <div className="info-grid">
+              <div className="image-container" onClick={handleOpenModal}>
+                <div className="image-content">
+                  <img
+                    src={nftState.value?.image}
+                    alt="token-logo"
+                    className="token-image"
+                    onError={handleOnError}
+                  />
+                  <img
+                    src={ZoomIcon}
+                    alt="zoom-icon"
+                    className="magnify-zoom-icon"
+                  />
+                </div>
+              </div>
+              <div className="details-container">
+                <div className="section-title">DETAILS</div>
+                <div className="description-block">
+                  <span className="title">DESCRIPTION</span>
+                  <span className="content">{nftState.value?.description}</span>
+                </div>
+                <div className="info-grid">
+                  <div className="content-block">
+                    <span className="title">CREATOR</span>
+                    <span className="grid-content">
+                      {nftState.value?.creatorName ||
+                        truncateText(true, nftState.value?.creatorAddress)}
+                    </span>
+                  </div>
+                  <div className="content-block">
+                    <span className="title">COLLECTION</span>
+                    <div className="grid-content">
+                      {/*
+                        TODO: uncomment the code below when Ghostmarket fix their API, currently is not returning an image for the collection
+                        <img
+                          className="collection-logo"
+                          src={nftState.value?.collection.image}
+                          alt="token-logo"
+                        />
+                      */}
+                      <span>{nftState.value?.collection.name}</span>
```

Comment:
vai dar erro se value ou collection for null 
```suggestion
                      <span>{nftState.value?.collection?.name}</span>
```